src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -75,55 +78,48 @@ def get_crash_free_historical_data(
     )
 
 
-def format_historical_data(data: SnubaTSResult, dataset: Any) -> list[TimeSeriesPoint]:
+def format_historical_data(
+    data: SnubaTSResult, query_columns: list[str], dataset: Any, organization: Organization
+) -> list[TimeSeriesPoint]:
     """
     Format Snuba data into the format the Seer API expects.
-    For errors/transactions data:
-        If there are no results, it's just the timestamp
-        {'time': 1719012000}, {'time': 1719018000}, {'time': 1719024000}
-
-        If there are results, the aggregate is added
-        {'time': 1721300400, 'count': 2}
-
-    For metrics_performance dataset/sessions data:
-        The count is stored separately from the timestamps, if there is no data the count is 0
     """
+    serializer = SnubaTSResultSerializer(organization=organization, lookup=None, user=None)
```

Comment:
I'm using the same serializer the `events-stats` endpoint uses and just pulling that data off to format into a list of `TimeSeriesPoint`s for Seer's API